keyboards/gmmk/pro/mcuconf.h

```diff
@@ -1,6 +1,273 @@
-#pragma once
+/*
```

Comment:
Please revert to the small file.  This should only be the settings that are changed from the default. 

You can use `qmk chibios-confmigrate` to generate a new version, if needed.

Reply:
Done